assets/js/modules/search-console/components/module/ModuleOverviewWidget/Stats.js

```diff
@@ -24,31 +24,19 @@ import PropTypes from 'prop-types';
 /**
  * WordPress dependencies
  */
-import { __, sprintf } from '@wordpress/i18n';
+import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
-import { CORE_SITE } from '../../../../../googlesitekit/datastore/site/constants';
-import { decodeHTMLEntity } from '../../../../../util';
 import { getSiteStatsDataForGoogleChart } from '../../../util';
 import { Grid, Row, Cell } from '../../../../../material-components';
 import GoogleChart from '../../../../../components/GoogleChart';
-import Data from 'googlesitekit-data';
-const { useSelect } = Data;
 
 const Stats = ( { data, metrics, selectedStats } ) => {
-	const currentEntityTitle = useSelect( ( select ) => select( CORE_SITE ).getCurrentEntityTitle() );
-
-	let title = __( 'Search Traffic Summary', 'google-site-kit' );
-	if ( currentEntityTitle ) {
-		/* translators: %s: page title */
-		title = sprintf( __( 'Search Traffic Summary for %s', 'google-site-kit' ), decodeHTMLEntity( currentEntityTitle ) );
-	}
-
 	const options = {
 		chart: {
-			title,
+			title: __( 'Search Traffic Summary', 'google-site-kit' ),
```

Comment:
The changes in this file are kind of unrelated, but I spotted this in merged code while noticing that this PR here originally handled the current entity which is generally unnecessary for module page widgets.